_data/meltano/loaders/target-mssql/storebrand.yml

```diff
@@ -0,0 +1,45 @@
+name: target-mssql
+variant: storebrand
+label: Microsoft SQL Server
+logo_url: /assets/logos/loaders/mssql.png
```

Comment:
I found https://images.app.goo.gl/szsXvxMyuBb29ErZ9 which seems to be the logo w/o the wordmark - can we use that?